db/migrate/20150618112155_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,11 @@
+class AddConfirmableToDevise < ActiveRecord::Migration
+  def change
+  	add_column :users, :confirmation_token, :string
+		add_column :users, :confirmed_at, :datetime
+		add_column :users, :confirmation_sent_at, :datetime
+		add_column :users, :unconfirmed_email, :string # Only if using reconfirmable
+
+		add_index :users, :confirmation_token, unique: true
+		execute("UPDATE users SET confirmed_at = NOW()")
```

Comment:
Inconsistent indentation detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Tab detected.